common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -139,6 +139,7 @@ def create(cls,
             else:
                 module.get_score = lambda: {'score': 0, 'total': 1}
 
+        module.graded = 'False'
```

Comment:
I'm concerned about adding the graded property here directly to the module, if that's not really the way it is done in practice. Can we add in the proper inheritance?